UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -826,6 +826,22 @@ private void SetConsciousState(ConsciousState NewConsciousState)
 			if (HealthIsLow()) OnLowHealth?.Invoke();
 		}
 
+
+
+		public BodyPart GetFirstBodyPartInArea(BodyPartType bodyPartAim)
+		{
+			foreach (var bodyPart in SurfaceBodyParts.ToArray())
+			{
+				if (bodyPart.BodyPartType == bodyPartAim)
+				{
+					return bodyPart;
+				}
+			}
+
+			return null;
```

Comment:
```suggestion
			return SurfaceBodyParts.FirstOrDefault(b => b.BodyPartType == bodyPartAim);
```

Reply:
like linq is nice but it's a pain to debug so idk